src/ui/libs/DatalensChartkit/ChartKit/plugins/Table/renderer/components/Table/Table.scss

```diff
@@ -256,6 +256,10 @@
         z-index: -1;
         overflow: auto;
 
+        table {
+            min-width: auto;
```

Comment:
Does this style make sense?

Reply:
Yes, in the table visible to the user, the minimum width is different - `min-width: 100%;`